src/lib/loadEventsNewsMetadataMap.ts

```diff
@@ -0,0 +1,100 @@
+import gql from "graphql-tag";
+import { print as printQuery } from "graphql";
+import type { ContentfulClient } from "$lib/services/server/contentful";
+import type { MetadataMapItem } from "./types";
+
+const queryNewsAndEvents = gql`
+  query NewsCollection {
+    newsCollection(where: { indexInSearch: true }, order: [publicationDate_DESC]) {
+      items {
+        sys {
+          id
+        }
+        type
+        title
+        subhead
+        publicationDate
+        slug
+        byline
+      }
+    }
+    eventEntryCollection(where: { indexInSearch: true }, order: [eventDateAndTime_ASC]) {
+      items {
+        sys {
+          id
+        }
+        slug
+        shortTitle
+        eventDescription
+        eventDateAndTime
+      }
+    }
+  }
+`;
+
+// generated query types are not returning
+type NewsAndEventsCollectionQuery = {
+  newsCollection: {
+    items: {
+      sys: { id: string };
+      __typename?: "News";
+      title?: string | null;
+      publicationDate?: string | null;
+      slug?: string | null;
+    }[];
+  };
+  eventEntryCollection: {
+    items: {
+      sys: {
+        id: string;
+      };
+      __typename?: "News";
+      title?: string | null;
+      slug?: string | null;
+      eventDateAndTime?: string | null;
+      eventDescription?: string | null;
+    }[];
+  };
+};
+
+// TODO: remove
+const constructEventSlug = (date: Date, slug: string): string =>
+  `${date.toISOString().split("T")[0]}-${slug}`;
```

Comment:
We should use the [new util](https://github.com/la-ldaf/ldaf-site/blob/b1b3602037af48cfa8291a7667f036aefcf4df27/src/lib/util/constructEventSlug.ts) instead.